docs/docsite/rst/porting_guides/porting_guide_core_2.19.rst

```diff
@@ -90,6 +612,7 @@ Plugins
 
      ansible_ssh_password_mechanism: sshpass
 
+
```

Comment:
Why the extra newline?

Reply:
Last line of a section where we've got 2 blank lines everywhere else in the file